<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SP2Open</class>
 <widget class="QDialog" name="SP2Open">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>2-Dimensional Spectrum</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>200</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTreeWidget" name="loctree">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Edit Locus</string>
     </property>
    </column>
   </widget>
   <widget class="QRadioButton" name="locadd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Add points (K)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="locdel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Exclude points (L)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Q W E
A  S D  - Keys to move marker
Z     C
Shift moves faster
X - hit point
V - take the polygon</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>23</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Drawing</string>
    </property>
    <widget class="QSlider" name="scale">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>30</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
    <widget class="QSlider" name="threshold">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QSlider" name="markersize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Marker size</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read source</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="left">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>580</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="right">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>580</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="down">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QToolButton" name="up">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="calbox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Calibration</string>
   </property>
   <widget class="QRadioButton" name="peak1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>peak1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="peak2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>peak2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="peak3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>peak3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Just move marker</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="e1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="e2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="e3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="splabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Color of spectrum</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>White</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
   </widget>
   <widget class="QSlider" name="colorslider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QSlider" name="brightness">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>48</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dull</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Selection colors</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Locus</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Marker</string>
    </property>
   </widget>
   <widget class="QSlider" name="LocCol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="MarkerCol">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
